### PR TITLE
convert all files to func and call them in main 

### PR DESCRIPTION
**Pull request by YuweAEC**
image

Invoke all functions in main
image

invoke once with determining each value

### CODE REVIEW

File: daily-code-update/day-8.c [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+void main()
+{
+    int a = 30;
+    if (ARANGE)
+        printf("within range");
+    else
+        printf("out of range");
+}
 
 #define FOUND printf("The Yankee Doodle Virus");
```
Comment (gauravsinghaec) [inline daily-code-update/day-8.c]: why is this macro is defined here? it should be in the beginning of program?

File: daily-code-update/day-8.c [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 
-// void main()
-// {
-//     int a = 30;
-//     if (ARANGE)
-//         printf("within range");
-//     else
-//         printf("out of range");
-// }
+void main()
```
Comment (gauravsinghaec) [inline daily-code-update/day-8.c]: why do you have two main function in one file?

File: daily-code-update/day-8.c (hunk 1 above)
Comment (YuweAEC) [reply daily-code-update/day-8.c]: okay re solving it

File: daily-code-update/day-8.c (hunk 2 above)
Comment (YuweAEC) [reply daily-code-update/day-8.c]: please check my last commit.
to see changes

File: daily-code-update/day-8.c (hunk 1 above)
Comment (YuweAEC) [reply daily-code-update/day-8.c]: please check functions in file changed once for final correction of code

**Review by gauravsinghaec — APPROVED** [review]

**Pull request merged**